packages/frontend/app/components/courses/root.js

```diff
@@ -64,7 +62,20 @@ export default class CoursesRootComponent extends Component {
     return this.allRelatedCoursesData.isResolved ? this.allRelatedCoursesData.value : null;
   }
 
-  @use canCreateCourse = new PermissionChecker(() => ['canCreateCourse', this.selectedSchool]);
+  @cached
+  get canCreateData() {
+    return new TrackedAsyncData(
+      this.selectedSchool
+        ? this.permissionChecker.canCreateCourse(this.selectedSchool)
+        : new Promise((resolve) => {
```

Comment:
`TrackedAsyncData` doesn't require a promise, you can just return `false` here and it will mark as `isResolved` and set that as the `value`

Reply:
nice! i started out with that but it blew up horribly, which i attributed to the lack of a Promise value on the falsy condition.
must have been something else...

anyway, i'm glad you checked up on this. 

handled :heavy_check_mark: